sorc/build_forecast.sh

```diff
@@ -23,5 +23,5 @@ if [ $target = jet ]; then target=jet.intel ; fi
 
 cd hafs_forecast.fd/
 cd tests/
-./compile.sh "$target" "CCPP=Y STATIC=Y SUITES=HAFS_v0_gfdlmp_nonsstugwd,HAFS_v0_gfdlmp_nocpnsstugwd,HAFS_v0_gfdlmp_nocpnsst,HAFS_v0_gfdlmp_nocp,HAFS_v0_gfdlmp_nougwd,HAFS_v0_gfdlmp_nocpugwd,HAFS_v0_gfdlmp,HAFS_v0_hwrf_thompson,HAFS_v0_hwrf_nougwd,HAFS_v0_hwrf 32BIT=Y HYCOM=Y" 32bit YES NO
+./compile.sh "$target" "CCPP=Y STATIC=Y SUITES=HAFS_v0_gfdlmp_nonsstugwd,HAFS_v0_gfdlmp_nocpnsstugwd,HAFS_v0_gfdlmp_nocpnsst,HAFS_v0_gfdlmp_nonsst,HAFS_v0_gfdlmp_nocp,HAFS_v0_gfdlmp_nougwd,HAFS_v0_gfdlmp_nocpugwd,HAFS_v0_gfdlmp,HAFS_v0_hwrf_thompson,HAFS_v0_hwrf_nougwd,HAFS_v0_hwrf 32BIT=Y HYCOM=Y CMEPS=Y" 32bit YES NO
```

Comment:
@uturuncoglu, The new CCPP suite of HAFS_v0_gfdlmp_nonsst is added so that the FV3ATM can run properly with the convection parameterization and gravity wave drag (GWD) parameterizations on, but turning off the NSST component when coupling with HYCOM of course. This is the new CCPP suite we will use for our future experiments.